src/main/java/ev3dev/actuators/lego/motors/BaseRegulatedMotor.java

```diff
@@ -206,16 +206,31 @@ public void hold() {
      * Cancels any rotate() orders in progress
      */
     public void stop() {
-		this.setStringAttribute(COMMAND, STOP);
-
-        for (RegulatedMotorListener listener : listenerList) {
-            listener.rotationStopped(this, this.getTachoCount(), this.isStalled(), System.currentTimeMillis());
-        }
+        stop(true);

```

Comment:
This is consistent with leJOS:
https://sourceforge.net/p/lejos/ev3/code/ci/master/tree/ev3classes/src/lejos/hardware/motor/BaseRegulatedMotor.java#l183
```
float speed = 0,
int acceleration = <acceleration>,
int limit = <no limit>,
boolean hold = TRUE,
boolean waitComplete = TRUE
```

Reply:
However there is the change mentioned in the ADR - waitComplete is false by default now.